stylus/objects/layouts.styl

```diff
@@ -40,6 +41,30 @@ $fullbleed
     left 0
     right 0
 
+// Elastic Layout — Usually with fixed header and/or fixed footer and an extensible content zone with scrolling inside
+
+               // linear-gradient(alpha(coolGrey,.2) 25%, alpha(white,0)),
+               // linear-gradient(alpha(white,0), alpha(coolGrey,.2) 75%) 0 100%,
+$elastic
+    display flex
+    flex-flow column nowrap
+    align-items stretch
+
+$elastic-content
+    background  linear-gradient(white 30%, alpha(black, 0)),
+                linear-gradient(alpha(black, 0), white 70%) 0 100%,
+                linear-gradient(alpha(coolGrey, .25) 0, alpha(coolGrey, .25) 25%, alpha(white, 0) 26%, alpha(white, 0) 100%),
+                linear-gradient(alpha(white, 0) 0, alpha(white, 0) 74%, alpha(coolGrey, .25) 75%, alpha(coolGrey, .25) 100%) 0 100%
+    background-repeat no-repeat
+    background-color white
+    background-size 100% 2rem, 100% 2rem, 100% .5rem, 100% .5rem
+    background-attachment local, local, scroll, scroll
```

Comment:
Can you leave a comment to say that this is used to show when the user has scrolled ?